packages/sw-cli/src/lib/templates/sw.js.tmpl

```diff
@@ -0,0 +1,6 @@
+importScripts(
+  "<%= manifestPath %>",
```

Comment:
Using a timestamp to trigger a SW update is overly aggressive. We want SW updates only when something our SW cares about changes, not whenever there's a build performed.

Granted, that doesn't introduce very much overhead, assuming our `install`/`activate` handlers are lightweight.

But more to your point, I understand that you want to have a model in which folks can edit the top-level service worker file, and I support that. I am confused as to *when* you think the top-level file should be edited, though. My understanding was that folks would edit the file prior to running it through the Rollup bundling step, not edit the file afterwards.

In your model, are folks taking the file that's been spit out by this build process, including the timestamp you propose, and then editing that? Won't their edits be lost the next time the file is generated?